pkgs/tools/misc/flameshot/default.nix

```diff
@@ -33,6 +33,6 @@ mkDerivation rec {
     homepage = "https://github.com/flameshot-org/flameshot";
     maintainers = with maintainers; [ scode oxalica ];
     license = licenses.gpl3Plus;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
This can be shortened to `platforms.unix`

Reply:
That will also include FreeBSD, but maybe that's fine :). People using those systems know they're kinda on their own anyway.

Reply:
I guess I explicitly just added Darwin since I had not tested on any of the BSD's. I'm fine with either way.